_events/2020-05-08.md

```diff
@@ -63,10 +63,23 @@ timetable:
 
 images:
   - /images/events/stuttgart/01.jpg
+  - /images/events/stuttgart/02.jpg
+  - /images/events/stuttgart/03.jpg
+  - /images/events/stuttgart/04.jpg
+  - /images/events/stuttgart/05.jpg
 
 redirect_from:
   - /stuttgart
 
 members:
+  - Pierre Neis
+  - Mariana Podesta
   - Christoph Reinicke
+  - Kerstin Biedermann
+  - Juergen Rambo
```

Comment:
Because there is no profile for this person, it will not show up as an organizer. There should always be a profile.